### PR TITLE
Flag `--cosign-artifacts` on describe command

### PR DESCRIPTION
**Pull request by joaopapereira**
Add a flag to do not retrieve cosign artifacts while describing bundle

### CODE REVIEW

File: pkg/imgpkg/cmd/describe.go [hunk 1]
```diff
@@ -51,6 +52,7 @@ func NewDescribeCmd(o *DescribeOptions) *cobra.Command {
 	o.RegistryFlags.Set(cmd)
 	cmd.Flags().IntVar(&o.Concurrency, "concurrency", 5, "Concurrency")
 	cmd.Flags().StringVarP(&o.OutputType, "output-type", "o", "text", "Type of output possible values: [text, yaml]")
+	cmd.Flags().BoolVar(&o.ExcludeCosignArtifacts, "exclude-cosign-artifacts", false, "Do not retrieve cosign artifact information")
```
Comment (cppforlife) [inline pkg/imgpkg/cmd/describe.go]: exclude-cosign-artifacts => cosign-artifacts=bool (in alignment with cosign-signatures=bool)

File: pkg/imgpkg/cmd/describe.go [hunk 2]
```diff
@@ -51,6 +52,7 @@ func NewDescribeCmd(o *DescribeOptions) *cobra.Command {
 	o.RegistryFlags.Set(cmd)
 	cmd.Flags().IntVar(&o.Concurrency, "concurrency", 5, "Concurrency")
 	cmd.Flags().StringVarP(&o.OutputType, "output-type", "o", "text", "Type of output possible values: [text, yaml]")
+	cmd.Flags().BoolVar(&o.IncludeCosignArtifacts, "cosign-artifacts", true, "Do not retrieve cosign artifact information (Default: true)")
```
Comment (cppforlife) [inline pkg/imgpkg/cmd/describe.go]: i think the description is flipped. it's including cosign-artifacts by default, right?

File: pkg/imgpkg/cmd/describe.go (hunk 1 above)
Comment (joaopapereira) [reply pkg/imgpkg/cmd/describe.go]: fixed

File: pkg/imgpkg/cmd/describe.go (hunk 2 above)
Comment (joaopapereira) [reply pkg/imgpkg/cmd/describe.go]: fixed

**Pull request merged**